src/protocols/secure_channel/CASESession.h

```diff
@@ -253,6 +254,7 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
     FabricTable * mFabricsTable    = nullptr;
     const FabricInfo * mFabricInfo = nullptr;
     NodeId mPeerNodeId             = kUndefinedNodeId;
+    NodeId mLocalNodeId            = kUndefinedNodeId;
```

Comment:
I guess one question I have is: why can't we just get the local node id from `mFabricInfo` as needed?   I guess mFabricInfo could go stale if a fabric is removed while we're in the middle of a CASE session establishment for that fabric, but then we have bigger problems with this code...

This is OK for now, though, and we should sort out the lifetimes of these things separately.

Reply:
If a PASE session for an admin does a RemoveFabric of a fabric in process of CASE, then the more direct deps we have later on access to fabric table (in between messages), the more problematic it becomes. This symmetry to storage of local and peer node made more sense to me